<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>988</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-30</y>
      <width>998</width>
      <height>998</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgba(2, 2, 13, 0.659)</string>
    </property>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>181</width>
       <height>511</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgba(2, 2, 13, 0.659)</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>181</width>
        <height>511</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="acc_info">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255,255,255);
border-radius : 10px</string>
         </property>
         <property name="text">
          <string>Tài khoản của tôi</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="trading_history">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255,255,255);
border-radius : 10px</string>
         </property>
         <property name="text">
          <string>Lịch sử giao dịch</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="add_money">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255,255,255);
border-radius : 10px</string>
         </property>
         <property name="text">
          <string>Nạp tiền</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="logout">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255,255,255);
border-radius : 10px</string>
         </property>
         <property name="text">
          <string>Đăng xuất </string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>90</y>
       <width>811</width>
       <height>511</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgba(2, 2, 13, 0.659)</string>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QLabel" name="user_name">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>60</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255,255,255)</string>
       </property>
       <property name="text">
        <string>Họ và tên</string>
       </property>
      </widget>
      <widget class="QLabel" name="user_age">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>130</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255,255,255)</string>
       </property>
       <property name="text">
        <string>Tuổi</string>
       </property>
      </widget>
      <widget class="QLabel" name="user_money">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>200</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255,255,255)</string>
       </property>
       <property name="text">
        <string>Số tiền</string>
       </property>
      </widget>
      <widget class="QLabel" name="user_name_2">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>60</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255,255,255)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="user_age_2">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>130</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255,255,255)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="user_money_2">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>200</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255,255,255)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="change_pw">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>290</y>
         <width>171</width>
         <height>40</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255,255,255);
border-radius : 10px</string>
       </property>
       <property name="text">
        <string>Đổi mật khẩu</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="change_info">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>380</y>
         <width>171</width>
         <height>40</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255,255,255);
border-radius : 10px</string>
       </property>
       <property name="text">
        <string>Thay đổi thông tin</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>811</width>
         <height>511</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>521</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background:rgba(2, 2, 13, 0.659)</string>
       </property>
       <column>
        <property name="text">
         <string>Tên phim</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Giá tiền</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Ngày mua</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3">
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>140</y>
         <width>291</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255,255,255)</string>
       </property>
       <property name="text">
        <string>Chọn mệnh giá muốn nạp</string>
       </property>
      </widget>
      <widget class="QComboBox" name="nt">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>220</y>
         <width>321</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255,255,255)</string>
       </property>
       <item>
        <property name="text">
         <string>50000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>100000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>200000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>500000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1000000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2000000</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>40</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255,255,255)</string>
       </property>
       <property name="text">
        <string>Số tiền hiện tại </string>
       </property>
      </widget>
      <widget class="QLabel" name="cur_money">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>30</y>
         <width>281</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255,255,255)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="nt_btn">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>320</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255,255,255);
border-radius : 10px</string>
       </property>
       <property name="text">
        <string>Nạp tiền</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>40</y>
       <width>251</width>
       <height>39</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255,0,0);
background-color: rgba(0,0,0,0)
</string>
     </property>
     <property name="text">
      <string>Thông tin cá nhân</string>
     </property>
    </widget>
    <widget class="QPushButton" name="back">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#back{
color :rgb(83, 78, 209);
border-radius : 5px
}
QPushButton#back:hover{
background-color: rgba(212, 181, 181, 0.729);
}</string>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
